src/models/__tests__/visitor.test.integration.ts

```diff
@@ -49,11 +49,16 @@ describe('Visitor model', () => {
       expect(exists).toBe(false);
     });
 
-    test('fromOrganisation :: gets all visitors at organisation', async () => {
```

Comment:
Was the `fromOrganisation` test deleted in the rebase? We should probably retrieve that.

Reply:
Hehe oh dear...